Hyperlint Automation: Meta Description Fixes (#20254)
cloudflare/cloudflare-docs: src/content/docs/browser-rendering/platform/wrangler.mdx

---
pcx_content_type: configuration
title: Wrangler

> description: Wrangler: Command-line tool for deploying projects using Cloudflare's Workers Browser Rendering API. Install, configure bindings, and deploy with ease.

Reviewer: ```suggestion
description: Use Wrangler, a command-line tool, to deploy projects using Cloudflare's Workers Browser Rendering API.
```
